modules/ROOT/pages/troubleshoot-access-management.adoc

```diff
@@ -0,0 +1,41 @@
+= Troubleshoot Access Management
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+These are some tips about what to do if you encounter these issues. 
+
+== Problems with Accessing Anypoint Platform
+
+=== Sign in Credentials are not recognized
+
+If your credentials aren't recognized when you try to sign into Anypoint Platform, try the following:
+
+* Check that you are using the correct URL to sign into your organization. +
+If your organization uses an external identity management system, your login URL is *anypoint.mulesoft.com/accounts/login/_yourOrganizationName_*.
```

Comment:
Domain, not Name

Reply:
This can be confusing since their organization can use external identity management but their specific user does not. So if their user does not, they continue to login at anypoiny.mulesoft.com/login/